cfme/tests/cloud/test_instance_power_control.py

```diff
@@ -25,7 +24,7 @@ def pytest_generate_tests(metafunc):
 def testing_instance(request, setup_provider, provider):
     """ Fixture to provision instance on the provider
     """
-    instance = Instance.factory(random_vm_name('pwr-ctrl'), provider)
+    instance = Instance.factory(random_vm_name('pwr-c'), provider)
```

Comment:
1st version was made right after 1st version of name generator - default  length less than 40 symbols.
and after name generator started giving 15symbols long  names by default - last part was cut anyway. so just to meet defaults